react-components/src/hooks/useSceneDefaultCamera.tsx

```diff
@@ -32,13 +32,42 @@ export const useSceneDefaultCamera = (
       };
     }
 
+    const cameraNotSet =
```

Comment:
Boolean values should be prefix with `is`, `should` or `has` in my opinion

Reply:
Usually, I would also prefer them to be positive. In this case, I would go for `isCameraSet`